pom.xml

```diff
@@ -184,7 +184,6 @@
       <groupId>org.apache.apex</groupId>
       <artifactId>apex-engine</artifactId>
       <version>${apex.core.version}</version>
-      <scope>test</scope>
```

Comment:
Hi @tweise 
Can you please take a look at incubator-apex-malhar/library/src/main/java/com/datatorrent/lib/util/AbstractKeyValueStorageAgent.java:[229,34]

and suggest what can be done to accommodate this change without changing test score of apex-engine.